content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/08-upgrade-from-mongodb-beta.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: 'Upgrade from MongoDB Beta'
+metaTitle: 'Upgrade from the Prisma 1 MongoDB Beta to Prisma 2'
+metaDescription: 'Learn how to upgrade your MongoDB application running Prisma 1 to Prisma 2.'
+---
+
+## Introduction
+
+This guide aims to help you migrate from the MongoDB Beta on Prisma 1 to MongoDB on Prisma 2. To learn more about the differences between Prisma 1 and Prisma 2, you can refer to [this document](https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/how-to-upgrade#main-differences-between-prisma-1-and-prisma-version-2x-and-later).
+
+The scope of this guide is to give you the workflow necessary to perform the migration and highlight some of the gotchas you may encounter.
+
+We unfortunately can't cover all possible scenarios or changes required, but this guide should help you on your journey. Join [our community Slack](https://slack.prisma.io/) or create an issue [on Github](https://github.com/prisma/prisma1/issues/new/choose) with any questions.
+
+Please perform this migration on your staging environment before trying this in production!
+
+## Requirements
+
+- Must be running MongoDB 4.2+ as a replica set (MongoDB Atlas does this for you automatically)
+- Node.js 12.6.x or 14.x or 16.x
+- TypeScript 4.1.x
+
+## Installing Prisma 2
+
+In your project directory run the following commands:
+
+```bash
+$ npm install prisma
```

Comment:
Needed as they could still have a `prisma` in their dependencies already, even pinned:
```suggestion
$ npm install prisma@latest
```